RoboticonColony/Assets/Code/Tile.cs

```diff
@@ -0,0 +1,86 @@
+﻿using UnityEngine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+sealed public class Tile
+{
+    public int Cost { get; private set; }
+    public AbstractPlayer Owner { get; private set; }
+    private Roboticon installedRoboticon { get; set; }
+    private Dictionary<ItemType, int> resources;
+
+    /// <summary>
+    /// Constructor for tile
+    /// </summary> 
+    /// <param name="cost">Cost of the tile</param>
+    /// <param name="ore">Ore that the tile has</param>
+    /// <param name="power">Power that the tile has</param>
+    public Tile(int cost, int ore, int power)
+    {
+        Cost = cost;
+        resources = new Dictionary<ItemType, int>();
+        resources[ItemType.Ore] = ore;
+        resources[ItemType.Power] = power;
+        Owner = null;
+        installedRoboticon = null;
+    }
+
+    /// <summary>
+    /// Adds a roboticon to the tile
+    /// </summary>
+    /// <param name="newRoboticon">Roboticon to be added</param>
+    public void AddRoboticon(Roboticon newRoboticon)
+    {
+        installedRoboticon = newRoboticon;
+    }
+
+    /// <summary>
+    /// Removes the roboticon from the tile
+    /// </summary>
+    public void RemoveRoboticon()
+    {
+        installedRoboticon = null;
+    }
+
+    public void AddPlayer(Player player)
```

Comment:
Now Owner is public and so is the setter, This is redundant and should be removed.